code/game/objects/random/cloth.dm

```diff
@@ -0,0 +1,180 @@
+//CLOTH RANDOM
+/obj/random/cloth/masks
+	name = "random mask"
+	desc = "This is a random mask."
+
+/obj/random/cloth/masks/item_to_spawn()
+	return pick(subtypesof(/obj/item/clothing/mask/))
```

Comment:
There exist few purevirtual types of masks (like one with AI vision). Also cigaretes is masks too (may be not a problem probably should be mentoined in comment here).

Reply:
I checked out the mask in question, thats pretty cool. I'm tempted to allow it and just say that kind of thing is a perk

however, it throws runtimes. so i'll exclude it